plugins/sync.js

```diff
@@ -533,22 +593,27 @@ window.plugin.sync.Authorizer = function(options) {
   this.isAuthorizing = this.isAuthorizing.bind(this);
   this.authorize = this.authorize.bind(this);
 };
-
+// values that are static at the moment
 window.plugin.sync.Authorizer.prototype.CLIENT_ID = '1099227387115-osrmhfh1i6dto7v7npk4dcpog1cnljtb.apps.googleusercontent.com';
 window.plugin.sync.Authorizer.prototype.SCOPES = 'https://www.googleapis.com/auth/drive.file';
 
+// method to tell if SYNC is authorized
 window.plugin.sync.Authorizer.prototype.isAuthed = function() {
   return this.authorized;
 };
 
+// method to tell if authorization is in progress
 window.plugin.sync.Authorizer.prototype.isAuthorizing = function() {
   return this.authorizing;
 };
+
+// (external) method to add an other authCallback function 
 window.plugin.sync.Authorizer.prototype.addAuthCallback = function(callback) {
   if(typeof(this.authCallback) === 'function') this.authCallback = [this.authCallback];
   this.authCallback.push(callback);
 };
 
+// (internal) function that will execute all enregistered authCallback functions 
```

Comment:
what is "internal" ? private? ... then maybe mark it the good old way: "_authComplete"